src/connector/src/test_data/proto_recursive/recursive.pb

```diff
@@ -0,0 +1,20 @@
+
+�
+recursive.proto	recursive"�
+ComplexRecursiveMessage
+	node_name (	RnodeName
+node_id (RnodeIdM
```

Comment:
Is this the compiled result? Might be better to git ignore it?

Reply:
the proto dep requires the compiled ver

Reply:
I mean can we compile it when testing? Seem we don't have any other test that depends on the hard coded compile output.

Reply:
Never mind, not a big deal. We can always handle it later.